winpr/libwinpr/synch/wait.c

```diff
@@ -179,9 +179,9 @@ static void ts_add_ms(struct timespec* ts, DWORD dwMilliseconds)
 
 DWORD WaitForSingleObjectEx(HANDLE hHandle, DWORD dwMilliseconds, BOOL bAlertable)
 {
-	ULONG Type;
-	WINPR_HANDLE* Object;
-	WINPR_POLL_SET pollset;
+	ULONG Type = 0;
+	WINPR_HANDLE* Object = NULL;
+	WINPR_POLL_SET pollset = { 0 };
```

Comment:
ok for pollset init (call of `pollset_uninit` at the end of the function), but not for `Type` and `Object` that are set just below by `winpr_Handle_GetInfo`

Reply:
@hardening `winpr_Handle_GetInfo` is a function that might change.